### PR TITLE
Modify Fancy Bed textures

### PR DESCRIPTION
**Pull request by TumeniNodes**
This covers the discussions in #2251

### CODE REVIEW

**Comment by paramat** [comment]
Texture has an unnecessary alpha channel, should be converted to indexed too.

**Comment by paramat** [comment]
Otherwise looks good.

File: mods/beds/README.txt [hunk 1]
```diff
@@ -11,6 +11,9 @@ Authors of media (textures)
 ---------------------------
 BlockMen (CC BY-SA 3.0)
```
[the remark this answers is not in the document]
Comment (TumeniNodes) [reply mods/beds/README.txt]: added

**Comment by paramat** [comment]
Will test.

**Comment by paramat** [comment]
screenshot_20181028_225041

Also we need to replace the default:wood texture for the simple bed underside, same texture for that is fine.

**Comment by TumeniNodes** [comment]
> Also we need to replace the default:wood texture for the simple bed underside, same texture for that is fine.

Can't believe I forgot to do that... I did it in my other test build. dur

**Comment by TumeniNodes** [comment]
If this is set, would be nice to see merged... as I made a bit of a woops.
If you notice, I did not create a new branch in my master to do this PR (you are allowed to laugh)
and there is another PR I would like to start working on, as soon as I get this mess fixed : D

**Comment by paramat** [comment]
Sorry, busy MT day.

**Comment by paramat** [comment]
:+1:

**Pull request merged**